ArcGISRuntimeSDKQt_CppSamples/Search/OfflineGeocode/OfflineGeocode.cpp

```diff
@@ -242,73 +284,50 @@ void OfflineGeocode::connectSignals()
     emit suggestInProgressChanged();
   });
 
-  // if clicked pin graphic, show callout. otherwise, reverse geocode
-  connect(m_mapView, &MapQuickView::identifyGraphicsOverlayCompleted, this, [this](const QUuid&, IdentifyGraphicsOverlayResult* rawIdentifyResult)
-  {
-    // Delete rawIdentifyResult when we leave scope.
-    auto identifyResult = std::unique_ptr<IdentifyGraphicsOverlayResult>(rawIdentifyResult);
-
-    if (!identifyResult)
-      return;
-
-    // if user clicked on pin, display callout
-    const QList<Graphic*> graphics = identifyResult->graphics();
-    if (graphics.count() > 0)
-      m_calloutData->setVisible(true);
-
-    // otherwise, reverse geocode at that point
-    else
-    {
-      m_isReverseGeocode = true;
-      m_locatorTask->reverseGeocodeWithParameters(m_clickedPoint, m_reverseGeocodeParameters);
+  connect(m_locatorTask, &LocatorTask::errorOccurred, this, &OfflineGeocode::logError);
```

Comment:
The new async functions will not emit errors via errorOccurred. You need new logic to capture the `onFailed` for those, and probably `logError` signature needs to be updated to take an ErrorException now.

Reply:
Done.